includes/class-scd-ext-manual-drip.php

```diff
@@ -180,7 +180,7 @@ public function update_manual_drip_activity() {
 			'action'     => 'update'
 		);
 
-		if ( 'Give Access' === $_POST[ 'scd_log_learner_lesson_manual_drip_submit' ] ) {
+		if ( __( 'Give Access', 'sensei-content-drip' ) === $_POST['scd_log_learner_lesson_manual_drip_submit'] ) {
```

Comment:
Is the POST variable translated as well? This code didn't seem to execute when I tried it.

Reply:
Yeah, because the button is an `input` element its `value` attribute needs to contain the text to be displayed. Because that text is translated, we also need to check for the translated text in the POST data.

Before I changed this line, clicking the `Give Access` button did not work properly.

Reply:
Oh, interesting. Here's what happened. My French translation is `Give l\'access`.

`__( 'Give Access', 'sensei-content-drip' )` returns "Give l'access"
`$_POST['scd_log_learner_lesson_manual_drip_submit']` returns "Give l\'access", so the code doesn't execute.

Reply:
But aren't those two strings both `Give l'access`? So shouldn't the `if` statement be `true`?

Reply:
Chatted on Slack. One of them is `Give l\'access`. I'll look into a better solution here.

Reply:
@donnapep I submitted a fix for this in 4c758ba